migrations/automatic/educandu-2021-12-10-01-add-import-related-indexes.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2021_12_10_01_add_import_related_indexes {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    await this.db.collection('tasks').createIndexes([
+      {
+        name: '_idx_batch_id_',
+        key: { batchId: 1 }
+      }
+    ]);
+  }
+
+  async down() {
+    await this.db.collection('tasks').dropIndex('_idx_batch_id_');
+  }
+}
```

Comment:
Should we consider
1. a combined index `tasks._id: taskId` + `tasks.processed` ? The combined query is used in `task-processor.js`
2. a combined index `tasks.processed` + `tasks.batchId`? The combined query is used in `batch-processor.js`

Though I'm not sure such indexes would bring much in these usage cases 🤷‍♀️

Reply:
We would have to measure it, but maybe that's a good idea, it might speed up task processing.

Reply:
Done